src/helpers/helpers.core.js

```diff
@@ -276,3 +276,7 @@ export function resolveObjectKey(obj, key) {
 	}
 	return obj;
 }
+
+export function ucFirst(str) {
```

Comment:
What do you think about calling this `capitalize`? I'd also lean towards prefixing it with an underscore

Reply:
done